docs/pywbemcli/generaloptions.rst

```diff
@@ -393,13 +393,24 @@ any subsequent pywbemcli commands:
 ``--timeout`` general option
 """"""""""""""""""""""""""""
 
-The argument value of the ``--timeout``/``-t`` general option is an integer that
-defines the
-client side timeout in seconds. The pywbem client includes a timeout mechanism
-that closes a WBEM connection if there is no response to a request to the WBEM
-server in the time defined by this value. Pywbemcli defaults to a
+The argument value of the ``--timeout``/``-t`` general option is an integer
+that defines the client side read timeout in seconds. The pywbem client
+includes a timeout mechanism that closes a WBEM connection if there is no
+response to a request to the WBEM server in the time defined by this value and
+the retry count.
+
+The option value is the timeout for a single read request to the WBEM server. Since
+pywbem retries the command multiple times (normally 2 retries), the time to
```

Comment:
I did not want to make it specific here because we cannot tie the value to the code itself.  Suggest we make specific in code and help and point to that from here.

Reply:
Modified this to point back to the help for this option for an actual number